pkg/render/guardian.go

```diff
@@ -368,6 +430,58 @@ func (c *GuardianComponent) annotations() map[string]string {
 	return annotations
 }
 
+func (c *GuardianComponent) deploymentSelector() *metav1.LabelSelector {
+	return &metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"app.kubernetes.io/name": GuardianDeploymentName,
```

Comment:
Maybe a constant for the app.kubernetes.io/name label is appropriate if we're using it in a bunch of places?